frontend/model/contracts/group.js

```diff
@@ -1742,7 +1763,8 @@ sbp('chelonia/defineContract', {
       //   1) automatically switch that user to a 'pledging' member with 0 contribution,
       //   2) pop out the prompt message notifying them of this automatic change,
       //   3) and send 'MINCOME_CHANGED' notification.
-      const myProfile = sbp('state/vuex/getters').ourGroupProfile
+      const identityContractID = sbp('state/vuex/state').loggedIn.identityContractID
+      const myProfile = sbp('chelonia/rootState')[contractID].profiles[identityContractID]
```

Comment:
How is `'state/vuex/state'` and `'chelonia/rootState'` different here? Can a comment be added to explain this?

Reply:
Based on our conversation about this, could you please add a comment above line 1766 to say something like this:

```js
// using 'state/vuex/state' here instead of 'chelonia/rootState' to fetch the identityContractID because although
// both 'chelonia/rootState' and 'state/vuex/state' are the same at the moment, in the future they will not be, and
// all calls to 'state/vuex/state' will need to be replaced in the future with something else. Using 'state/vuex/state'
// here now makes it easier to find these calls in the future.
```